www/site/content/docs/tools-reference/berks.md

```diff
@@ -0,0 +1,12 @@
++++
+title = "Tools Reference - berks"
```

Comment:
Seems overkill to have `Tools Reference` on each page when that will be clearly indicated by the Nav

Reply:
I went back and forth on that - what I actually wanted was to get the full name in the <head> title, "Chef Workstation Tools Reference - TOOLNAME", while the menu item and the page heading just have the tool name. 
I'll push the update to be just the tool name, and can open another card to straighten out titles on the browser titlebar (which will be only  TOOLNAME with that change), vs what's shown on the menu and in the content content.

Reply:
Updated, new screencaps inbound